fuel-rxjava/src/main/kotlin/com/github/kittinunf/fuel/rx/RxFuel.kt

```diff
@@ -24,15 +24,15 @@ private fun <T : Any> Request.rx_response(deserializable: Deserializable<T>): Ob
                 when (result) {
```

Comment:
Aha. I would love to use this opportunity not using `.onError`, but using only `onNext`. Also, I think it might be just better to use `Single<Pair<Response, T>>`. What do you think?